js/src/apbct-public--1--functions.js

```diff
@@ -8,7 +16,13 @@ function ctSetCookie( cookies, value, expires ){
     // Cookies disabled
     if( ctPublicFunctions.data__cookies_type === 'none' ){
         cookies.forEach( function (item, i, arr	) {
-           apbctLocalStorage.set(item[0], encodeURIComponent(item[1]))
+            if ( typeof force_alternative_method_for_cookies !== 'undefined' ) {
+                if (force_alternative_method_for_cookies.indexOf(item[0]) !== -1) {
+                    ctSetAlternativeCookie(cookies)
+                } else {
```

Comment:
Need to be aggregated to a single ajax request

Reply:
Moved to the outer place. However, there is no effect because of the current cookie setting implementation. We need to think about aggregation in the future.